docs/setup/compute-ntp.md

```diff
@@ -40,3 +45,9 @@ If your Create® 3 and compute board have an internet connection, this is not re
         ===============================================================================
         192.168.186.2                  51      0   5   -    32       0      0   -     -
         localhost                       0      0   -   -     -      31      0   7     4
+
+1. Note that if there is a large jump in the time, the Create® 3 may not accept it until its next reboot.
+    This can be verified by checking the Create® 3 robot's log for a line like
+
+        user.notice ntpd: ntpd: reply from 192.168.186.3: delay ### is too high, ignoring
+    If this happens, simply restart the robot (not just the application) via the webserver over the USB network connection.
```

Comment:
Do we have a curl call for this restart from SBC?

Reply:
I can generate one, but I would rather not, until we have a defined API (otherwise, if the webserver changes, the call will change)